### PR TITLE
Resizing arg gis maps

### PR DESCRIPTION
**Pull request by rubyimrie**

### CODE REVIEW

File: pages/council.vue [hunk 1]
```diff
@@ -12,8 +12,35 @@
       </div>
 
     </div>
-    <div>
-      <style>.embed-container {position: relative; padding-bottom: 71%; height: 0; max-width: 100%;} .embed-container iframe, .embed-container object, .embed-container iframe{position: absolute; top: 0; left: 0; width: 100%; height: 100%;} small{position: absolute; z-index: 40; bottom: 0; margin-bottom: -15px;}</style><div class="embed-container"><iframe width="900" height="640" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" title="Choropleth Map of Trial Location" src="//edinuniv.maps.arcgis.com/apps/Embed/index.html?webmap=18afc056bff34ae3a7d76fade8cd195d&extent=-5.6973,55.3553,-1.6406,56.674&zoom=true&previewImage=false&scale=true&search=true&searchextent=true&legend=true&disable_scroll=false&theme=dark"></iframe></div>
+    <div style="height: 90%;">
```
Comment (Sephster) [inline pages/council.vue:15]: Has anything changed here or is this just a code format change? Not spotting the edits. Sorry if they are there!

File: pages/council.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div style="height: 100%;">
```
Comment (Sephster) [inline pages/council.vue:2]: Is this the main change? I'm not seeing it affecting anything on my screen but wondering if I perhaps don't have the right resolution to show the issue. Are you able to give a bit of context for this if this is the crux of the change? Thank you

**Review by Sephster — CHANGES_REQUESTED** [review]

File: pages/council.vue (hunk 2 above)
Comment (rubyimrie) [reply pages/council.vue:2]: This is how it was looking before on mobile devices before adding this. It didn't take up the full space available.
image
This how it now looks on mobile devices, taking up the full screen.
image

File: pages/council.vue (hunk 1 above)
Comment (rubyimrie) [reply pages/council.vue:15]: No edit here sorry!

Comment (Sephster) [reply pages/council.vue:15]: Cool, thanks for confirming Ruby. Marking as resolved

File: static/images/Maggie.png
Comment (Sephster) [inline static/images/Maggie.png:1]: These images have been added in but don't seem to be part of the PR's intended reason. Just checking if these have been added in in error or should maybe be in a different PR. Thanks Ruby

**Review by Sephster — CHANGES_REQUESTED** [review]

File: static/images/Maggie.png
Comment (rubyimrie) [reply static/images/Maggie.png:1]: Yes sorry these shouldn't of been in this PR. I have removed them now.

Comment (rubyimrie) [reply static/images/Maggie.png:1]: Yes sorry these images shouldn't of been in this PR. I have removed them now.
